static/.htaccess

```diff
@@ -180,6 +180,19 @@ RewriteRule ^docs/next/apis-tools/tasklist-api/(.*)$ /docs/next/apis-tools/camun
 #   8.7: content moves introduced prior to the release of version 8.7.
 #---------------------------------------------------------------------------------
 
+# Reroute element templates and HTTL to C7 docs
+RewriteRule ^docs/components/modeler/desktop-modeler/element-templates/c7-defining-templates/(.*)$ https://docs.camunda.org/manual/latest/modeler/element-templates/ [R=301,L]
+RewriteRule ^docs/next/components/modeler/desktop-modeler/element-templates/c7-defining-templates/(.*)$ https://docs.camunda.org/manual/latest/modeler/element-templates/ [R=301,L]
```

Comment:
@christinaausley It looks that the `next` redirection does not work properly. The redirection kicks in only when I remove `next` from the path.

![image](https://github.com/user-attachments/assets/95a2ed51-1e27-4cb5-a63c-3edb59a13f2c)

Reply:
I will launch a PR @barmac 👍 This likely happened with the release.

Reply:
Is there a reason the /next redirects need to exist? We consider the vNext docs to be in-progress. The only time we preserve URLs in vNext with a redirect is when it is required by an external link from the product or connector element templates.

Reply:
Hmm perhaps we don't need that? We don't usually link to `next`, so my manual try could be considered as an edge case. Sorry for unnecessary trouble.

Reply:
I was wondering this @pepopowitz -- I can remove them 👍